README.md

```diff
@@ -39,8 +51,14 @@ make
 Finally, build and run the example app:
 
 ```
-cd examples/hello
-pipenv install
-spin build
+cd examples/hello_world
+../../target/release/spin-python app -o app.wasm
 spin up
 ```
+
+**Note:* In the samples provided in the `examples` directory, the `spin build` command depends on the plugin `py2wasm` being installed. Therefore, to test the locally built `spin-python` binary, either replace the build command in the `spin.toml` to invoke it using spin build.
+
+```
+[component.build]
+command = "../../target/release/spin-python py2wasm app - o app.wasm"
```

Comment:
Had trouble with this part. When I made this change (using `-o` instead of `- o`) I received the following error:

```
gitpod /workspace/spin-python-sdk/examples/hello_world (main) $ spin build
Executing the build command for component python-sdk-example: ../../target/release/spin-python py2wasm app -o app.wasm
error: unexpected argument 'app' found

Usage: spin-python [OPTIONS] <APP_NAME>

For more information, try '--help'.
Error: Build command for component python-sdk-example failed with status Exited(2)
gitpod /workspace/spin-python-sdk/examples/hello_world (main) $ 
```

However I was able to make `spin build` work by using the following instead:
`command = "../../target/release/spin-python app -o app.wasm"`

Reply:
Yes that is the intended command. My apologies for the typo. Thanks once again for bringing this up.